controller/src/main/java/org/jboss/as/controller/OperationContext.java

```diff
@@ -418,6 +418,30 @@
      */
     String getCurrentAddressValue();
 
+    /**
+     * Get the node with current operation name
+     *
+     * @return operation name node
+     */
+    ModelNode getCurrentOperationName();
+
+    /**
+     * Get parameter node by its name
+     *
+     * @param name of desired parameter
+     * @return node for parameter of given name
+     */
+    ModelNode getCurrentOperationParameter(String name);
+
+    /**
+     * Get parameter node by its name
+     *
+     * @param name of desired parameter
+     * @param nullable whether return value can be null
```

Comment:
whether the return value can be null if the operation has no parameter with the given name